watcher/tool/watcher/main.go

```diff
@@ -25,14 +25,21 @@ import (
 	"github.com/gravitational/monitoring-app/watcher/lib/kubernetes"
 
 	"github.com/gravitational/trace"
+	"github.com/gravitational/version"
 	log "github.com/sirupsen/logrus"
 )
 
 func main() {
 	var mode string
 	flag.StringVar(&mode, "mode", "", fmt.Sprintf("watcher mode: %v", constants.AllModes))
+	ver := flag.Bool("version", false, "print current watcher version")
```

Comment:
I think `print version` would suffice - it cannot be anything other than `current` at this point.